### PR TITLE
Ion Laws Get Modified

### PR DESCRIPTION
**Pull request by TheReddDragon**
# General Documentation

### Intent of your Pull Request
adds some more variety to ion laws

### Why is this change good for the game?
council said to remove some plus I wanted funny haha crew doesnt require shoes meme

# Wiki Documentation
### What should players be aware of when it comes to the changes your PR is implementing?
more ion laws can form
### What general grouping does this PR fall under? 
AI ION LAWS

# Changelog

:cl:  
rscadd: Added some new varieties to ion law lists  
rscdel: Removed some things @TehLadyK told me to
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
Not so sure about more meta player references tbh

**Comment by UselessTheremin** [comment]
Especially in ion laws, with how it will force the AI to do something with them. What are they gonna do, find John sweeps?

**Comment by Marmio64** [comment]
blatant player references should be avoided and you didn't even include a great list of yog players

File: strings/yogs_ion_laws.json [hunk 1]
```diff
@@ -138,7 +138,11 @@
 		"TRESPASSERS",
 		"BALD PEOPLE",
 		"HAIRY PEOPLE",
-		"MOTHS"
+		"MOTHS",
+        "JOHN SWEEPS",
+        "JOESPH PRAT",
+        "JEFF JEFFERSON",
+        "VENUS"
```
Comment (alexkar598) [inline strings/yogs_ion_laws.json]: 😕

**Comment by UselessTheremin** [comment]
I’m gonna make my own ion law PR, with blackjack and hookers
FCE7E39A-C2C8-4F65-BA92-B117326C2A09

File: strings/yogs_ion_laws.json (hunk 1 above)
Comment (TheReddDragon) [reply strings/yogs_ion_laws.json]: removed meta references

**Comment by TheReddDragon** [comment]
@UselessTheremin removed meta player references

**Comment by Hopekz** [comment]
Could probably reopen your PR after your changes since it is difficult to see what people think about your foot fetish law without the player part

**Comment by JamieD1** [comment]
Reopen it so we can get a better opinion

**Pull request closed without merging**